tests/read.rs

```diff
@@ -24,7 +29,7 @@ fn read_rss090() {
     assert_eq!(channel.link(), "http://www.mozilla.org");
     assert_eq!(
         channel.description(),
-        "the Mozilla Organization\n      web site"
+        format!("the Mozilla Organization{}      web site", LINE_ENDING)
```

Comment:
Can we just use [`lines()`](https://doc.rust-lang.org/stable/std/primitive.str.html#method.lines) here?

Reply:
unfortunately:
```
error[E0369]: binary operation `==` cannot be applied to type `std::str::Lines<'_>`
```

Reply:
right iterator...